programs/stake/src/stake_state.rs

```diff
@@ -969,6 +1007,17 @@ pub fn calculate_points(
     }
 }
 
+// utility function, used by Split
```

Comment:
maybe add a comment saying the context: `doing here because not to break compatibility; newer stake program ABI can just use newly-required Rent sysvar` or the like.

Reply:
Added